src/main/java/frc/robot/subsystems/Dashboard.java

```diff
@@ -0,0 +1,335 @@
+import frc.robot.Constants;
+import frc.robot.subsystems.climb.Climb;
+import frc.robot.subsystems.drive.Drive;
+import frc.robot.subsystems.intake.Intake;
+import frc.robot.subsystems.notepath.Notepath;
+import frc.robot.subsystems.shooter.ShooterBed;
+import frc.robot.subsystems.shooter.ShooterFlywheel;
+
+public class Dashboard extends SubsystemBase
+{
+    private static Dashboard _instance;
+    private Field2d          _field;
+
+    // Widgets for the dashboard
+    private GenericEntry          _allianceBox;
+    private GenericEntry          _HasNote;
+    private GenericEntry          _frAngle;
+    private GenericEntry          _flAngle;
+    private GenericEntry          _brAngle;
+    private GenericEntry          _blAngle;
+    private GenericEntry          _LeftHeight;
+    private GenericEntry          _Rightheight;
+    private GenericEntry          _IntakeSpeed;
+    private GenericEntry          _notepathOutput;
+    private GenericEntry          _bedAngle;
+    private GenericEntry          _upperVelocity;
+    private GenericEntry          _lowerVelocity;
+    private GenericEntry          _frOffset;
+    private GenericEntry          _flOffset;
+    private GenericEntry          _brOffset;
+    private GenericEntry          _blOffset;
+    private GenericEntry          _IntakeInSpeed;
+    private GenericEntry          _climberleftOffset;
+    private GenericEntry          _climberRightOffset;
+    private GenericEntry          _climberMinHeight;
+    private GenericEntry          _NoteShootSpeed;
+    private GenericEntry          _PickupIntakeSpeed;
+    private GenericEntry          _ShooterIntakeSpeed;
+    private GenericEntry          _ShooterOffset;
+    private GenericEntry          _IntakeOutSpeed;
+    private GenericEntry          _bedMinimumAngle;
+    private GenericEntry          _bedMaximumAngle;
+    private GenericEntry          _maxFlywheelVelocity;
+    private GenericEntry          _climberMaxHeight;
+    private GenericEntry          _maxFlywheelSpeed;
+    private GenericEntry          _FlywheelIntakeSpeed;
+    private final ShooterBed      _ShooterBed;
+    private final Notepath        _notepath;
+    private final Intake          _Intake;
+    private final ShooterFlywheel _ShooterFlywheel;
+    private final Drive           _drive;
+    private final Climb           _climb;
+
+    // SendableChoosers for Autonomous options
+    private SendableChooser<Integer> _autoDelayChooser;
+
+    public Dashboard(ShooterBed shooterBed, Notepath notepath, ShooterFlywheel ShooterFlywheel, Drive drive, Intake intake, Climb climb)
+    {
+        _drive           = drive;
+        _ShooterBed      = shooterBed;
+        _notepath        = notepath;
+        _ShooterFlywheel = ShooterFlywheel;
+        _Intake          = intake;
+        _climb           = climb;
+
+        var tab         = Shuffleboard.getTab("Dashboard");
+        var SettingsTab = Shuffleboard.getTab("Settings");
```

Comment:
After you move this, don't forget to make it camelCase:

```suggestion
        var settingsTab = Shuffleboard.getTab("Settings");
```